server/src/main/java/org/eclipse/hono/server/HonoServer.java

```diff
@@ -380,51 +337,16 @@ public boolean isOpenSecurePort() {
      *
      * @return the insecure port Hono listens on.
      */
-    public int getInsecurePort() {
-        if (honoConfig.isInsecurePortUnconfigured() && !honoConfig.isInsecurePortEnabled())
+    public int getHonoServerInsecurePort() {
```

Comment:
`getActualInsecurePort()`? final
why is this not implemented along the lines of `getHonoServerPort()`?